package.json

```diff
@@ -42,7 +42,8 @@
     "rollup": "^1.27.0",
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-babel-minify": "^9.1.1",
-    "rollup-plugin-node-resolve": "^5.2.0"
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-postcss": "^2.0.3"
```

Comment:
perhaps should consider https://github.com/RJHwang/rollup-plugin-css-porter

Reply:
<img width="186" alt="image" src="https://user-images.githubusercontent.com/9112811/69016406-473bd980-095b-11ea-86bb-144ebce1de7a.png">
How do I access this CSS hmm

Reply:
I thought it would work using
```js
import 'bandicoot/dist/bandicoot.esm.css'
```
https://github.com/rollup/rollup-plugin-node-resolve/issues/9#issuecomment-167595950

Reply:
Error: No PostCSS Config found in: /Users/devin/Code/Canopy/bandicoot/dist
Ah I'm wondering if I am getting this because it's not in node_modules and so my webpack config is trying to rebundle/load bandicoot's css

Reply:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/9112811/69016684-ee217500-095d-11ea-8596-0c45155c6f0d.png">

Reply:
I can get it with the consumer to run with 
<img width="251" alt="image" src="https://user-images.githubusercontent.com/9112811/69017024-1e1e4780-0961-11ea-80cd-c9d070e5c296.png">
but I'm getting a bunch of `You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing.` and check out this class obfuscation:
<img width="403" alt="image" src="https://user-images.githubusercontent.com/9112811/69017021-0e9efe80-0961-11ea-9a8f-539b6214d624.png">

Reply:
Hmm, even if I do this from bandicoot rollup config:
```js
  {
    input: 'postcss.config.js',
    output: {
      file: 'dist/postcss.config.js',
      format: 'esm',
    }
  }
```
it's a no go. @joeldenning any guidance you can give me on how to `require the user to import a single .css file`

Reply:
I think it would be even better if the user of bandicoot didn't even have to require the user to import a single css file. The reason is that it's often hard for users to actually import the CSS file successfully and get it downloaded and stuff in their webpack bundles. Especially if they're using css-modules where it interprets all css as local, or if they're using emotion/styled-components where they aren't even using css-loader at all and would have to introduce it solely to use the bandicoot css.

I prefer this option: "could be manually appending a style element"

^ it's better because it requires no configuration by the user.

Reply:
I'm in agreement but I finally got the style export to work. Had to adjust webpack of consumer to NOT use postcss loader on the .esm.css file. That combined with having the random `postcss.config.js` made things work..

I've updated for manual DOM insertion